tests/test_chebyshevBoundedField.py

```diff
@@ -89,6 +89,23 @@ def setUp(self):
     def tearDown(self):
         del self.bbox
 
+    def testFillImageInterpolation(self):
+        ctrl, coefficients = self.cases[-2]
+        bbox = lsst.afw.geom.Box2I(lsst.afw.geom.Point2I(10, 15),
+                                   lsst.afw.geom.Extent2I(360, 350))
+        field = lsst.afw.math.ChebyshevBoundedField(bbox, coefficients)
+        image1 = lsst.afw.image.ImageF(bbox)
+        image2 = lsst.afw.image.ImageF(bbox)
+        image3 = lsst.afw.image.ImageF(bbox)
+        image4 = lsst.afw.image.ImageF(bbox)
+        field.fillImage(image1)
+        field.fillImage(image2, xStep=3)
+        field.fillImage(image3, yStep=4)
+        field.fillImage(image4, xStep=3, yStep=4)
+        self.assertFloatsAlmostEqual(image1.array, image2.array, rtol=2E-2, atol=2E-2)
```

Comment:
Shouldn't the tolerances for these each be different, since the step sizes are different?